pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -92,6 +92,12 @@ type ScrapeConfigSpec struct {
 	// MetricsPath HTTP path to scrape for metrics. If empty, Prometheus uses the default value (e.g. /metrics).
 	// +optional
 	MetricsPath *string `json:"metricsPath,omitempty"`
+	// Interval between consecutive scrapes.
+	// +optional
+	ScrapeInterval *v1.Duration `json:"scrapeInterval,omitempty"`
+	// Number of seconds to wait until a scrape request times out.
+	// +optional
+	ScrapeTimeout *v1.Duration `json:"scrapeTimeout,omitempty"`
```

Comment:
```suggestion
	// ScrapeInterval is the interval between consecutive scrapes.
	// +optional
	ScrapeInterval *v1.Duration `json:"scrapeInterval,omitempty"`
	// ScrapeTimeout is the number of seconds to wait until a scrape request times out.
	// +optional
	ScrapeTimeout *v1.Duration `json:"scrapeTimeout,omitempty"`
```

Reply:
Just to make sure godoc can read them correctly :)

Reply:
done